components/SelectedArticleScreen.jsx

```diff
@@ -1,15 +1,20 @@
 import { useParams } from 'react-router-dom';
-import styles from "./HomeScreen.module.css";
+import { useState } from "react";
+import styles from "./SelectedArticleScreen.module.css";
+import Comments from "./sub-components/Comments"
 import SelectedArticle from './sub-components/SelectedArticle';
 import NavBar2 from "./sub-components/NavBar2";
 import Header from "./sub-components/Header";
 
-const SelectedArticleScreen = ({votes, loading, setLoading}) => {
+const SelectedArticleScreen = ({votes, isLoading, setIsLoading}) => {
     const { article_id } = useParams()
+    const [comments, setComments] = useState([]);
```

Comment:
I disagree with the state for comments being here and set in a lower component to come back up. If you wish to conditionally render your comments that is great, however I'd argue the article component has nothing to do with comments and shouldn't be where the api call is happening.